### DISCUSSION TITLE
获取数据上传地址 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1227] [Announcements]
# 获取数据上传地址 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/insight/06UserGuide/01quickstart/gethosturl

**Comment by liyuerich** [comment]
其中，

lb-vminsert-insight-victoria-metrics-k8s-stack：指标数据上传地址
lb-insight-opentelemetry-collector：链路数据上传地址
mcamel-es-cluster-masters-es-http：日志数据上传地址

问题：还有个审计日志的地址是？我理解和日志地址一样？

**Comment by liyuerich** [comment]
选择集群 kpanda-global-cluster，选择左侧导航栏的容器应用 -> 服务

应该是容器网络->服务？

**Comment by ModetaNiu** [comment]
目前有了新的页面，见 https://docs.daocloud.io/insight/quickstart/install/gethosturl/
Thanks～
